raising-jiras.md

```diff
@@ -0,0 +1,87 @@
+#Raising Jiras
+
+<p>Jiras are used at Holiday Extras as a means for teams to communicate what work is required (including data and insights). This doc seeks to outline some general guidelines and best practice when writing a jira.</p>
+
+##Title:
+<p>This is a brief description of the problem to be solved. Be clear and to the point.</p>
+
+**Good example:**
+<p>"Change the endpoints in HAPI to new Horizon service for Flexible Car Hire"</p>
```

Comment:
Perhaps this isn't the best example? This jira was raised as a result of conversations that Ollie Buck had with Steve at Flexible, so they had already worked out the 'how/what'. Happy to use a different example for best practice.

I think this flags an interesting point around the knowns/unknowns of a piece of work - you can be much more specific where you have done a lot of background work/investigation already.